.github/workflows/lint-rust/action.yml

```diff
@@ -42,7 +41,7 @@ runs:
 
         - run: |
               cargo update
-              cargo install --locked --version 0.15.1 cargo-deny
+              cargo install --locked cargo-deny
```

Comment:
Version was fixed intetionally in https://github.com/valkey-io/valkey-glide/pull/2081. Please double check that it works now.

Reply:
Checked, it was a workaround instead of solution